scaffolder-templates/parasol-java-template/skeleton/.vscode/settings.json

```diff
@@ -1,10 +1,9 @@
 {
     "jest": {
         "nodeEnv": {
-            "NODE_TLS_REJECT_UNAUTHORIZED": "0", 
-            "PARASOL_MODEL_URL": "https://parasol-instruct-predictor-aiworkshop${{values.cluster}}:443/v1/chat/completions",
-            "EMBEDDING_MODEL_URL": "https://parasol-embedding-predictor-aiworkshop${{values.cluster}}/v1",
-            "PARASOL_MODEL_NAME": "parasol-instruct"
+            "PARASOL_MODEL_URL": "http://parasol-chat-trained-predictor.aiworkshop.svc.cluster.local:8080/v1/chat/completions",
```

Comment:
Shouldn't these have the `${{values.cluster}}` which is read from the helm values?

Reply:
Since all models are deployed as the RawDeployment, there’s no Route URL. Instead you need to access the model via service URL. You can find the example in the application.properties